models/sources.yml

```
@@ -7,12 +7,12 @@ sources:
       - name: events_20210131
       
   - name: google_sheets
-    schema: google_sheets
+    schema: module2
```

why not rename name in big query?
```suggestion
    schema: google_sheets
```